frontend/public/vendor.scss

```diff
@@ -29,9 +29,11 @@
 @import "~bootstrap-sass/assets/stylesheets/bootstrap/utilities";
 @import "~bootstrap-sass/assets/stylesheets/bootstrap/responsive-utilities";
 @import "~bootstrap-sass/assets/stylesheets/bootstrap/popovers";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/dropdowns";
```

Comment:
Mind adding a comment like I've done in https://github.com/openshift/console/pull/2728/files

Reply:
sure, done